predict-taxi-trip-duration/NYC Taxi 3 - Simple Featuretools.ipynb

```diff
@@ -229,7 +233,7 @@
     "<h2>Step 3: Create baseline features using DFS </h2>\n",
     "<p>Instead of manually creating features, such as month of <b>pickup_datetime</b>, we can let featuretools come up with them. </p>\n",
     "\n",
-    "<p>Within featuretools there is a standard format for representing data that is used to set up predictions and build features. A <b>EntitySet</b> stores information about entities (database table), variables (columns in database tables), relationships, and the data itself. </p>\n",
+    "<p>Within featuretools there is a standard format for representing data that is used to set up predictions and build features. A <b>EntitySet</b> stores information about dataframes (database table), columns in the dataframes, relationships, and the data itself. </p>\n",
```

Comment:
The wording of how entity sets are explained feels a little off to me. How about 

> An EntitySet stores typing information about the dataframe in the EntitySet, information about individual columns within each dataframe, the relationship between dataframes, and the data itself.

Reply:
@tamargrey  Yeah, that was a little clunky. Taking what you wrote and thinking about the definition, I came up with this alternative. The multiple use "dataframe" combined with "the data itself" didn't feel quite right to me. What do you think about this?

> An <b>EntitySet</b> stores one or more data tables (dataframes), typing information for the columns in each of the tables and the parent-child relationships between the tables in the EntitySet.

Reply:
I like that a lot!